CommunityToolkit.Graph.Uwp/CommunityToolkit.Graph.Uwp.csproj

```diff
@@ -22,7 +22,7 @@
   </ItemGroup>
     
 	<ItemGroup>
-    <PackageReference Include="Microsoft.Graph" Version="3.31.0" />
+    <PackageReference Include="Microsoft.Graph" Version="4.0.0" />
```

Comment:
I guess we should stay up-to-date. Does this need to be changed in the other graph packages too for alignment?

Too bad they pulled in the Azure Core, at least it updates to System.Text.Json...?

Reply:
Ya, just trying to stay current. I updated both of the Graph dependent packages (`CommunityToolkit.Graph` and `CommunityToolkit.Graph.Uwp`)